Etg.Yams/Storage/IYamsRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Threading.Tasks;
+using Etg.Yams.Application;
+using Etg.Yams.Storage.Config;
+
+namespace Etg.Yams.Storage
+{
+    public interface IYamsRepository
```

Comment:
Both names sound good to me. If we want a tie-breaker, I vote `IDeploymentRepository`